### PR TITLE
fix: spanish translation suggestions

### PR DESCRIPTION
**Pull request by cajotafer**
Hello, I see the spanish version was updated a few days ago, so I decided to contribute taking a look at it.

The changes are mostly small typos and other words updates that, I believe, match better with the context and intention of the original text.

### CODE REVIEW

**Comment by cajotafer** [comment]
Hello, I didn't tag you before @SachaG , I think this PR was forgotten 🐣

**Review by fransyrcc — CHANGES_REQUESTED** [review]
Hi @cajotafer thanks for the PR, apologies it took me a while to be able to check this, would it be possible to fix the following?

File: accounts.yml [hunk 1]
```diff
@@ -19,9 +19,10 @@ translations:
     - key: accounts.enter_username_or_email
       t: Introduce tu nombre de usuario o tu correo electrónico
     - key: accounts.error_internal_server_error
-      t: Error interno
+      t: Error de servidor
     - key: accounts.error_token_expired
-      t: Token inválido para renovar tu contraseña
+      t: Token de reestablecimiento de contraseña inválido
+      Invalid password reset token
```
[the remark this answers is not in the document]
Comment (cajotafer) [reply accounts.yml]: you're right, I have removed that line

File: common.yml [hunk 2]
```diff
@@ -137,7 +137,7 @@ translations:
 
           Tu información queda guardada. Puedes revisarla o modificarla hasta que la encuesta esté cerrada.
 
-          También puedes ayudarnos a llegar a todo el mundo compartiendo esta encuesta. Todo ayuda es poca. Así podremos hacer
+          También puedes ayudarnos a llegar a todo el mundo compartiendo esta encuesta. Todo ayuda cuenta. Así podremos hacer
```
[the remark this answers is not in the document]
Comment (cajotafer) [reply common.yml]: Sure, makes sense :D

File: results.yml [hunk 3]
```diff
@@ -741,13 +741,13 @@ translations:
   - key: share.email
     t: Enviar por correo
   - key: share.image
-    t: Conseguir imágen
+    t: Obtener imágen
```
[the remark this answers is not in the document]
Comment (cajotafer) [reply results.yml]: Oh, a typo while fixing typos. Solved

[the remark this answers is not in the document]
Comment (fransyrcc) [reply results.yml]: Hola @cajotafer, can you leave `imagen` without diacritic please? Other than this small one. All looks great!

[the remark this answers is not in the document]
Comment (cajotafer) [reply results.yml]: Hi @fransyrcc yep, this has been updated in the last commit as suggested. The version you commented here is outdated.

File: results.yml [hunk 4]
```diff
@@ -741,7 +741,7 @@ translations:
   - key: share.email
     t: Enviar por correo
   - key: share.image
-    t: Obtener imágen
+    t: Obtener imagen
```
Comment (cajotafer) [inline results.yml:744]: @fransyrcc

**Pull request merged**